app-arch/tar/tar-1.29.recipe

```diff
@@ -26,17 +26,33 @@ PROVIDES="
 REQUIRES="
 	haiku
 	"
+
 BUILD_REQUIRES="
 	haiku_devel
 	"
 BUILD_PREREQUIRES="
 	cmd:automake
 	cmd:autoreconf
+	cmd:find
 	cmd:gcc
 	cmd:ld
 	cmd:make
 	cmd:sed
 	"
+if [ "$recipeAction" = TEST ]; then
+	BUILD_PREREQUIRES="$BUILD_PREREQUIRES
```

Comment:
Well, I'd rather have a default TEST(), and TEST_PREREQUIRES inheriting BUILD_PREREQUIRES.
Conditional generally doesn't make a recipe more readable, on the contrary.
I also prefer to make a change in mass, and double check it, instead of going through tens of PR :)